crates/stc_ts_file_analyzer/src/analyzer/control_flow.rs

```diff
@@ -875,6 +875,7 @@ impl Analyzer<'_, '_> {
                                     return Err(Error::NotVariable {
                                         span: i.id.span,
                                         left: lhs.span(),
+                                        ty: Some(box ty.normalize().clone()),
```

Comment:
Please change this to `ty.clone()`

Reply:
Well, how do I have to type `Error::NotVariable`?
Currently the `ty` is Option<Box<Type>>, it makes conflict with this changes.

Reply:
fyi. ty is `Cow<Type>` in this file.

Reply:
Ah, I meant removing `normalize()` only

Reply:
Actually we use `Type::Arc` to avoid cloning and `normalize` dereferences `Type::Arc` into actual data type. So we should not call `.normalize().clone()` because it will clone actual data instead of `Arc`

Reply:
Oh.. `Cow` is problematic at the moment, I'll optimize this in future.  Thank you!
Anyway, please change match to call `normalize()` as it will guarantee that it'll work in future

Reply:
Thanks! `normalize()` at the use-side, right? I'll change it. @kdy1

Reply:
Yup